### ISSUE TITLE
WebGL backend for 2D Scatter?

### ISSUE THREAD

**Issue by JiaweiZhuang** [santosjorge/cufflinks#101]
For a large number of data points (>10000), the default 2D Scatter method is very slow. `Scattergl` solves this problem (https://plot.ly/python/webgl-vs-svg/)

Would be nice to have a `kind='scattergl'` option. The syntax of `Scattergl` seems almost the same as `Scatter`.

**Comment by gideonshalev** [comment]
Yes, please get this in soon as plotting lots of points is becoming more and more the norm.

**Comment by LocusBio** [comment]
Any progress on this? `kind='scatter'` is way to slow.
